### ISSUE TITLE
[QUESTION] "version" of the nodes

### ISSUE THREAD

**Issue by vincent-pli** [FlowiseAI/Flowise#1121]
I saw there is "version" in the definition of  `node`, some of them is `1.0` and others is `2.0`, what these versions do there?

**Comment by vincent-pli** [comment]
@HenryHengZJ

**Comment by HenryHengZJ** [comment]
when there is an update on the node input/output parameters, we update the version. This will allow UI to pick it up and notify user that a newer version is available

**Comment by vincent-pli** [comment]
Just for notification?
If user use latest version to `build` a original version node, could it backward compatibility？

**Comment by HenryHengZJ** [comment]
Yep just for notification, its not backward compatibility unfortunately
